lib/spells/effects/LocationEffect.h

```diff
@@ -23,8 +23,7 @@ namespace effects
 class LocationEffect : public Effect
 {
 public:
-	LocationEffect();
-	virtual ~LocationEffect();
+	virtual ~LocationEffect() override = default;
```

Comment:
same points about destructor

Reply:
@kambala-decapitator about removal - it can break qualifiers. By default destructor is not public AFAIK. But if destructor is here, it remain public. We need test if we need a public destructors of spell effects or not.

Reply:
missing destructor is equivalent to having public default destructor

> it can break qualifiers

sorry, I'm not following. Which qualifiers?

Reply:
@kambala-decapitator if public, you are right. I thougt than removed member of class is protected.

Reply:
@kambala-decapitator removed